.dev_scripts/batch_test_list.py

```diff
@@ -27,8 +27,8 @@
     dict(
         config='configs/cascade_rcnn/cascade_mask_rcnn_r50_fpn_1x_coco.py',
         checkpoint='cascade_mask_rcnn_r50_fpn_1x_coco_20200203-9d4dcb24.pth',
-        eval='bbox',
-        metric=dict(bbox_mAP=41.2),
+        eval='bbox segm',
```

Comment:
['bbox', 'segm'] would be better?

Reply:
If it is set to ['bbox', 'segm'], the code needs to be parsed, which will be more troublesome.